typhon/display.py

```diff
@@ -99,6 +99,22 @@ def display_type(self, value):
             self._display_type = value
             self.load_template(macros=self._last_macros)
 
+    @Property(str, designable=False)
+    def device_class(self):
+        """Full class with module name of loaded device"""
+        if self.devices:
+            device_class = self.devices[0].__class__
+            return '.'.join((device_class.__module__,
+                             device_class.__name__))
+        return ''
```

Comment:
Is it possible to do something like:
```
@Property(type, designable=False):
    if self.devices:
        return self.devices[0].__class__
    return object
```
To send literally the class? Then it would be easy to do `isinstance` and `issubclass` as needed, rather than just matching strings. Though I wouldn't be surprised if `pyqt` prohibits this.

Reply:
I could do that but how do you then compare that in a stylesheet?

Reply:
Just make stylesheets `.py` files

... just kidding. But maybe you have to do some import and `isinstance` magic either way, independent of swapping this.

Reply:
There has to be a way... hmm...